<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="20" passed="5" failed="2" skipped="13">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-06T14:52:13 IST" name="Suite" finished-at="2024-03-06T14:53:23 IST" duration-ms="70174">
    <groups>
      <group name="regressionTest">
        <method signature="TC_04_findingHospitals.verifyingPhoneFieldUsingInvalidMobileNumberFormat()[pri:7, instance:testClasses.TC_04_findingHospitals@3270d194]" name="verifyingPhoneFieldUsingInvalidMobileNumberFormat" class="testClasses.TC_04_findingHospitals"/>
        <method signature="TC_04_findingHospitals.verifyingPhoneFieldUsingBVA()[pri:8, instance:testClasses.TC_04_findingHospitals@3270d194]" name="verifyingPhoneFieldUsingBVA" class="testClasses.TC_04_findingHospitals"/>
        <method signature="TC_04_findingHospitals.verifyingPhoneFieldUsingBVA_1()[pri:9, instance:testClasses.TC_04_findingHospitals@3270d194]" name="verifyingPhoneFieldUsingBVA_1" class="testClasses.TC_04_findingHospitals"/>
        <method signature="TC_05_findingHospitals.verifyingMobileNumUsingValidInput()[pri:10, instance:testClasses.TC_05_findingHospitals@235834f2]" name="verifyingMobileNumUsingValidInput" class="testClasses.TC_05_findingHospitals"/>
        <method signature="TC_05_findingHospitals.verifyingMobileNumUsingValidInput_1()[pri:11, instance:testClasses.TC_05_findingHospitals@235834f2]" name="verifyingMobileNumUsingValidInput_1" class="testClasses.TC_05_findingHospitals"/>
        <method signature="TC_05_findingHospitals.verifyingMobileNumUsingValidInput_2()[pri:12, instance:testClasses.TC_05_findingHospitals@235834f2]" name="verifyingMobileNumUsingValidInput_2" class="testClasses.TC_05_findingHospitals"/>
        <method signature="TC_05_findingHospitals.verifyingMobileNumUsingValidInput_3()[pri:13, instance:testClasses.TC_05_findingHospitals@235834f2]" name="verifyingMobileNumUsingValidInput_3" class="testClasses.TC_05_findingHospitals"/>
        <method signature="TC_06_findingHospitals.verifyingEmailUsingInvalidInput()[pri:14, instance:testClasses.TC_06_findingHospitals@5656be13]" name="verifyingEmailUsingInvalidInput" class="testClasses.TC_06_findingHospitals"/>
        <method signature="TC_06_findingHospitals.verifyingEmailUsingInvalidInput_1()[pri:15, instance:testClasses.TC_06_findingHospitals@5656be13]" name="verifyingEmailUsingInvalidInput_1" class="testClasses.TC_06_findingHospitals"/>
        <method signature="TC_06_findingHospitals.verifyingEmailUsingInvalidInput_2()[pri:16, instance:testClasses.TC_06_findingHospitals@5656be13]" name="verifyingEmailUsingInvalidInput_2" class="testClasses.TC_06_findingHospitals"/>
        <method signature="TC_06_findingHospitals.verifyingEmailUsingInvalidInput_3()[pri:17, instance:testClasses.TC_06_findingHospitals@5656be13]" name="verifyingEmailUsingInvalidInput_3" class="testClasses.TC_06_findingHospitals"/>
        <method signature="TC_06_findingHospitals.verifyingEmailUsingValidInput()[pri:18, instance:testClasses.TC_06_findingHospitals@5656be13]" name="verifyingEmailUsingValidInput" class="testClasses.TC_06_findingHospitals"/>
        <method signature="TC_07_findingHospitals.verifyScheduleADemoButtonVisibility()[pri:20, instance:testClasses.TC_07_findingHospitals@4218d6a3]" name="verifyScheduleADemoButtonVisibility" class="testClasses.TC_07_findingHospitals"/>
      </group> <!-- regressionTest -->
      <group name="smokeTest">
        <method signature="TC_01_findingHospitals.search()[pri:0, instance:testClasses.TC_01_findingHospitals@4d5d943d]" name="search" class="testClasses.TC_01_findingHospitals"/>
        <method signature="TC_01_findingHospitals.filter()[pri:1, instance:testClasses.TC_01_findingHospitals@4d5d943d]" name="filter" class="testClasses.TC_01_findingHospitals"/>
        <method signature="TC_01_findingHospitals.getDoctorsData()[pri:2, instance:testClasses.TC_01_findingHospitals@4d5d943d]" name="getDoctorsData" class="testClasses.TC_01_findingHospitals"/>
        <method signature="TC_02_findingHospitals.navigateToSurgeries()[pri:3, instance:testClasses.TC_02_findingHospitals@55141def]" name="navigateToSurgeries" class="testClasses.TC_02_findingHospitals"/>
        <method signature="TC_02_findingHospitals.getSurgeriesList()[pri:4, instance:testClasses.TC_02_findingHospitals@55141def]" name="getSurgeriesList" class="testClasses.TC_02_findingHospitals"/>
        <method signature="TC_03_findingHospitals.navigateToCorporate()[pri:5, instance:testClasses.TC_03_findingHospitals@55182842]" name="navigateToCorporate" class="testClasses.TC_03_findingHospitals"/>
        <method signature="TC_03_findingHospitals.navigateToDemo()[pri:6, instance:testClasses.TC_03_findingHospitals@55182842]" name="navigateToDemo" class="testClasses.TC_03_findingHospitals"/>
      </group> <!-- smokeTest -->
    </groups>
    <test started-at="2024-03-06T14:52:13 IST" name="Test" finished-at="2024-03-06T14:53:23 IST" duration-ms="70174">
      <class name="testClasses.TC_06_findingHospitals">
        <test-method signature="verifyingEmailUsingInvalidInput_1()[pri:15, instance:testClasses.TC_06_findingHospitals@5656be13]" started-at="2024-03-06T14:53:23 IST" name="verifyingEmailUsingInvalidInput_1" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput_1()[pri:15, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput_1()[pri:15, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingEmailUsingInvalidInput_1 -->
        <test-method signature="verifyingEmailUsingInvalidInput()[pri:14, instance:testClasses.TC_06_findingHospitals@5656be13]" started-at="2024-03-06T14:53:23 IST" name="verifyingEmailUsingInvalidInput" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput()[pri:14, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput()[pri:14, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingEmailUsingInvalidInput -->
        <test-method signature="verifyingEmailUsingInvalidInput_2()[pri:16, instance:testClasses.TC_06_findingHospitals@5656be13]" started-at="2024-03-06T14:53:23 IST" name="verifyingEmailUsingInvalidInput_2" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput_2()[pri:16, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput_2()[pri:16, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingEmailUsingInvalidInput_2 -->
        <test-method signature="verifyingEmailUsingInvalidInput_3()[pri:17, instance:testClasses.TC_06_findingHospitals@5656be13]" started-at="2024-03-06T14:53:23 IST" name="verifyingEmailUsingInvalidInput_3" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput_3()[pri:17, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_06_findingHospitals.verifyingEmailUsingInvalidInput_3()[pri:17, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingEmailUsingInvalidInput_3 -->
        <test-method signature="verifyingEmailUsingValidInput()[pri:18, instance:testClasses.TC_06_findingHospitals@5656be13]" started-at="2024-03-06T14:53:23 IST" name="verifyingEmailUsingValidInput" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_06_findingHospitals.verifyingEmailUsingValidInput()[pri:18, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_06_findingHospitals.verifyingEmailUsingValidInput()[pri:18, instance:testClasses.TC_06_findingHospitals@5656be13] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingEmailUsingValidInput -->
      </class> <!-- testClasses.TC_06_findingHospitals -->
      <class name="testClasses.TC_01_findingHospitals">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:testClasses.TC_01_findingHospitals@4d5d943d]" started-at="2024-03-06T14:52:14 IST" name="setUp" finished-at="2024-03-06T14:52:20 IST" duration-ms="5944" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[user_input]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="search()[pri:0, instance:testClasses.TC_01_findingHospitals@4d5d943d]" started-at="2024-03-06T14:52:20 IST" name="search" finished-at="2024-03-06T14:52:21 IST" duration-ms="706" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- search -->
        <test-method signature="filter()[pri:1, instance:testClasses.TC_01_findingHospitals@4d5d943d]" started-at="2024-03-06T14:52:21 IST" name="filter" finished-at="2024-03-06T14:52:36 IST" duration-ms="14728" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- filter -->
        <test-method signature="getDoctorsData()[pri:2, instance:testClasses.TC_01_findingHospitals@4d5d943d]" started-at="2024-03-06T14:52:36 IST" name="getDoctorsData" finished-at="2024-03-06T14:52:37 IST" duration-ms="1142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDoctorsData -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:testClasses.TC_01_findingHospitals@4d5d943d]" started-at="2024-03-06T14:53:23 IST" name="afterClass" finished-at="2024-03-06T14:53:23 IST" duration-ms="702" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- testClasses.TC_01_findingHospitals -->
      <class name="testClasses.TC_05_findingHospitals">
        <test-method signature="verifyingMobileNumUsingValidInput()[pri:10, instance:testClasses.TC_05_findingHospitals@235834f2]" started-at="2024-03-06T14:53:23 IST" name="verifyingMobileNumUsingValidInput" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput()[pri:10, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput()[pri:10, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingMobileNumUsingValidInput -->
        <test-method signature="verifyingMobileNumUsingValidInput_1()[pri:11, instance:testClasses.TC_05_findingHospitals@235834f2]" started-at="2024-03-06T14:53:23 IST" name="verifyingMobileNumUsingValidInput_1" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput_1()[pri:11, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput_1()[pri:11, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingMobileNumUsingValidInput_1 -->
        <test-method signature="verifyingMobileNumUsingValidInput_2()[pri:12, instance:testClasses.TC_05_findingHospitals@235834f2]" started-at="2024-03-06T14:53:23 IST" name="verifyingMobileNumUsingValidInput_2" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput_2()[pri:12, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput_2()[pri:12, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingMobileNumUsingValidInput_2 -->
        <test-method signature="verifyingMobileNumUsingValidInput_3()[pri:13, instance:testClasses.TC_05_findingHospitals@235834f2]" started-at="2024-03-06T14:53:23 IST" name="verifyingMobileNumUsingValidInput_3" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput_3()[pri:13, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_05_findingHospitals.verifyingMobileNumUsingValidInput_3()[pri:13, instance:testClasses.TC_05_findingHospitals@235834f2] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingMobileNumUsingValidInput_3 -->
      </class> <!-- testClasses.TC_05_findingHospitals -->
      <class name="testClasses.TC_07_findingHospitals">
        <test-method signature="verifyScheduleADemoButtonVisibility()[pri:20, instance:testClasses.TC_07_findingHospitals@4218d6a3]" started-at="2024-03-06T14:53:23 IST" name="verifyScheduleADemoButtonVisibility" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_07_findingHospitals.verifyScheduleADemoButtonVisibility()[pri:20, instance:testClasses.TC_07_findingHospitals@4218d6a3] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_07_findingHospitals.verifyScheduleADemoButtonVisibility()[pri:20, instance:testClasses.TC_07_findingHospitals@4218d6a3] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyScheduleADemoButtonVisibility -->
      </class> <!-- testClasses.TC_07_findingHospitals -->
      <class name="testClasses.TC_04_findingHospitals">
        <test-method signature="verifyingPhoneFieldUsingInvalidMobileNumberFormat()[pri:7, instance:testClasses.TC_04_findingHospitals@3270d194]" started-at="2024-03-06T14:53:23 IST" name="verifyingPhoneFieldUsingInvalidMobileNumberFormat" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_04_findingHospitals.verifyingPhoneFieldUsingInvalidMobileNumberFormat()[pri:7, instance:testClasses.TC_04_findingHospitals@3270d194] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_04_findingHospitals.verifyingPhoneFieldUsingInvalidMobileNumberFormat()[pri:7, instance:testClasses.TC_04_findingHospitals@3270d194] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingPhoneFieldUsingInvalidMobileNumberFormat -->
        <test-method signature="verifyingPhoneFieldUsingBVA()[pri:8, instance:testClasses.TC_04_findingHospitals@3270d194]" started-at="2024-03-06T14:53:23 IST" name="verifyingPhoneFieldUsingBVA" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_04_findingHospitals.verifyingPhoneFieldUsingBVA()[pri:8, instance:testClasses.TC_04_findingHospitals@3270d194] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_04_findingHospitals.verifyingPhoneFieldUsingBVA()[pri:8, instance:testClasses.TC_04_findingHospitals@3270d194] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingPhoneFieldUsingBVA -->
        <test-method signature="verifyingPhoneFieldUsingBVA_1()[pri:9, instance:testClasses.TC_04_findingHospitals@3270d194]" started-at="2024-03-06T14:53:23 IST" name="verifyingPhoneFieldUsingBVA_1" depends-on-groups="smokeTest" finished-at="2024-03-06T14:53:23 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_04_findingHospitals.verifyingPhoneFieldUsingBVA_1()[pri:9, instance:testClasses.TC_04_findingHospitals@3270d194] depends on not successfully finished methods in group "smokeTest"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_04_findingHospitals.verifyingPhoneFieldUsingBVA_1()[pri:9, instance:testClasses.TC_04_findingHospitals@3270d194] depends on not successfully finished methods in group "smokeTest"
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyingPhoneFieldUsingBVA_1 -->
      </class> <!-- testClasses.TC_04_findingHospitals -->
      <class name="testClasses.TC_03_findingHospitals">
        <test-method signature="navigateToCorporate()[pri:5, instance:testClasses.TC_03_findingHospitals@55182842]" started-at="2024-03-06T14:53:18 IST" name="navigateToCorporate" finished-at="2024-03-06T14:53:18 IST" duration-ms="77" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToCorporate -->
        <test-method signature="navigateToDemo()[pri:6, instance:testClasses.TC_03_findingHospitals@55182842]" started-at="2024-03-06T14:53:18 IST" name="navigateToDemo" finished-at="2024-03-06T14:53:22 IST" duration-ms="4546" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToDemo -->
      </class> <!-- testClasses.TC_03_findingHospitals -->
      <class name="testClasses.TC_02_findingHospitals">
        <test-method signature="navigateToSurgeries()[pri:3, instance:testClasses.TC_02_findingHospitals@55141def]" started-at="2024-03-06T14:52:37 IST" name="navigateToSurgeries" finished-at="2024-03-06T14:52:57 IST" duration-ms="20057" status="FAIL">
          <exception class="org.openqa.selenium.ElementNotInteractableException">
            <message>
              <![CDATA[element not interactable
  (Session info: chrome=120.0.6099.130)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [fb279a3f3a5dc3155d7b1c946a61ba87, clickElement {id=4DD95A0A295D6ABEBDC249C0C7D66BDB_element_170}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:64814}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:64814/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (fb279a3f3a5dc3155d7b1c946a61ba87)] -> xpath: //*[text()='Surgeries']]
Session ID: fb279a3f3a5dc3155d7b1c946a61ba87]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementNotInteractableException: element not interactable
  (Session info: chrome=120.0.6099.130)
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [fb279a3f3a5dc3155d7b1c946a61ba87, clickElement {id=4DD95A0A295D6ABEBDC249C0C7D66BDB_element_170}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:64814}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:64814/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (fb279a3f3a5dc3155d7b1c946a61ba87)] -> xpath: //*[text()='Surgeries']]
Session ID: fb279a3f3a5dc3155d7b1c946a61ba87
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:200)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:175)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:607)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy46.click(Unknown Source)
at POM_files.doctors.surgeriesClick(doctors.java:94)
at testClasses.TC_02_findingHospitals.navigateToSurgeries(TC_02_findingHospitals.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementNotInteractableException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToSurgeries -->
        <test-method signature="getSurgeriesList()[pri:4, instance:testClasses.TC_02_findingHospitals@55141def]" started-at="2024-03-06T14:52:58 IST" name="getSurgeriesList" finished-at="2024-03-06T14:53:18 IST" duration-ms="20077" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//*[text()='Popular Surgeries']"}
  (Session info: chrome=120.0.6099.130)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [fb279a3f3a5dc3155d7b1c946a61ba87, findElement {using=xpath, value=//*[text()='Popular Surgeries']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:64814}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:64814/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: fb279a3f3a5dc3155d7b1c946a61ba87]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[text()='Popular Surgeries']"}
  (Session info: chrome=120.0.6099.130)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.15.0', revision: '1d14b5521b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [fb279a3f3a5dc3155d7b1c946a61ba87, findElement {using=xpath, value=//*[text()='Popular Surgeries']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:64814}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:64814/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: fb279a3f3a5dc3155d7b1c946a61ba87
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:200)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:175)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:607)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:364)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:358)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy46.getWrappedElement(Unknown Source)
at org.openqa.selenium.remote.internal.WebElementToJsonConverter.apply(WebElementToJsonConverter.java:51)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:470)
at POM_files.surgeries.scrollDown(surgeries.java:25)
at testClasses.TC_02_findingHospitals.getSurgeriesList(TC_02_findingHospitals.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getSurgeriesList -->
      </class> <!-- testClasses.TC_02_findingHospitals -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
